src/Library/demos/Toggle Button/main.blp

```diff
@@ -0,0 +1,89 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Toggle Button";
+  description: _("Represent active-state visually");
+
+  Box {
+    orientation: vertical;
+
+    Box Grouped_Toggle_Buttons {
+      orientation: horizontal;
+      halign: center;
+      margin-bottom: 12;
```

Comment:
please use 6px spacing for things that are relevant to each others like the label and its content